src/utils/quickPickWorkspaceFolder.ts

```diff
@@ -21,7 +22,12 @@ export async function quickPickWorkspaceFolder(context: IActionContext, noWorksp
         context.errorHandling.buttons = [
             {
                 callback: async () => {
-                    void vscode.commands.executeCommand('workbench.action.files.openFolder');
+                    if (isMac()) {
```

Comment:
So VS Code kept open https://github.com/microsoft/vscode/issues/120752 and indicated that `openFolder` might be registered on the Mac in future. Do you think we should revert to doing things uniformly on all platforms once that happens?

Reply:
That's a good question; I think the answer depends on whether `openFolder` on Mac forces opening _only_ folders, or if you can still open files. If it does enforce it, I definitely want to switch, in order to consistently enforce that folders must be opened. If it is not enforced--like "Open..."--maybe not.

Reply:
OK. I will let you decide whether and how you want to track this

Reply:
Sounds good. I'm subscribed to the upstream issue ('cause I opened it), so I'll see it if they close it fixed/won't-fix.